userguide/tasks.rst

```diff
@@ -794,6 +794,9 @@ describes the fields in this screen.
    | End                | drop-down menu             | Hour and minute the system must stop creating snapshots. Snapshots already in progress will continue until   |
    |                    |                            | complete.                                                                                                    |
    +--------------------+----------------------------+--------------------------------------------------------------------------------------------------------------+
+   | Allow taking empty | checkbox                   | Creates dataset snapshots when there are no changes. Set to support periodic snapshot schedules and          |
```

Comment:
The label for this field should be in title case.

Reply:
https://github.com/freenas/webui/pull/2455